set.mm

```diff
@@ -424505,130 +424571,196 @@ then they have the same degree (i.e., the same number of adjacent
-    frgrwopreg.e $e |- E = ( Edg ` G ) $.
-    $( Lemma 4 for ~ frgrwopreg .  In a friendship graph each vertex with
-       degree ` K ` is connected with a vertex with degree other than ` K ` .
-       This corresponds to statement 4 in [Huneke] p. 2:  "By the first claim,
-       every vertex in A is adjacent to every vertex in B.".  (Contributed by
-       Alexander van der Vekens, 30-Dec-2017.)  (Revised by AV, 10-May-2021.)
-       (Proof shortened by AV, 2-Jan-2022.) $)
-    frgrwopreglem4 $p |- ( G e. FriendGraph
-                           -> A. a e. A A. b e. B { a , b } e. E ) $=
-      ( vy wcel cv wceq wi cfrgr cpr cfv wne frgrwopreglem3 cnbgr wnel csn cdif
-      wa co wral frgrncvvdeq weq wb oveq2 neleq2 syl fveq2 eqeq1d neleq1 eqeq2d
-      imbi12d crab elrabi eleq2s adantr sneq difeq2d adantl eleq2i eldifeldifsn
-      sylan2b rspc2vd cusgr frgrusgr nbusgreledg prcom eleq1i syl6bb biimpa a1d
-      wn nnel expcom sylbi eqneqall 2a1d com12 syld com3l mpcom mpdi ralrimivv
-      ja ) FUAQZIRZJRZUBZEQZIJBCWPWQBQZWRCQZUJZWQDUCZWRDUCZUDZWTABCDFGHWQWRKLMN
-      UEPRZFARZUFUKZUGZXHDUCZXGDUCZSZTZPHXHUHZUIZULAHULZWPXCXFWTTZTAPDFHKLUMXCX
-      QWPXRXCXQWRFWQUFUKZUGZXDXESZTZWPXRTZXCYBXGXSUGZXDXLSZTXNAPWQWRHXPHWQUHZUI
-      ZAIUNZXJYDXMYEYHXIXSSXJYDUOXHWQFUFUPXIXSXGUQURYHXKXDXLXHWQDUSUTVCPJUNZYDX
-      TYEYAXGWRXSVAYIXLXEXDXGWRDUSVBVCXAWQHQZXBYJWQXKGSZAHVDBYKAWQHVEMVFVGYHXPY
-      GSXCYHXOYFHXHWQVHVIVJXBXAWRHBUIZQWRYGQCYLWRNVKBHWQWRVLVMVNYBXCYCXTYAXCYCT
-      ZXTWCWRXSQZYMWRXSWDYNYCXCWPYNXRWPYNUJWTXFWPYNWTWPYNWRWQUBZEQZWTWPFVOQYNYP
-      UOFVPEFWQWROVQURYOWSEWRWQVRVSVTWAWBWEWBWFYAXRXCWPWTXDXEWGWHWOWIWJWKWLWMWN
-      $.
-
-    $d A a x y z $.  $d B a b y z $.  $d E x z $.  $d G z $.
-    $( Lemma 5 for ~ frgrwopreg .  If ` A ` as well as ` B ` contain at least
-       two vertices, there is a 4-cycle in a friendship graph.  This
-       corresponds to statement 6 in [Huneke] p. 2:  "... otherwise, there are
-       two different vertices in A, and they have two common neighbors in
-       B, ...".  (Contributed by Alexander van der Vekens, 31-Dec-2017.)
-       (Revised by AV, 3-Jan-2022.) $)
-    frgrwopreglem5 $p |- ( ( G e. FriendGraph
-                              /\ 1 < ( # ` A ) /\ 1 < ( # ` B ) )
+    ${
+      $d A b $.  $d B x $.  $d D y $.  $d G a b x y $.  $d V y $.
+      frgrwopreg.e $e |- E = ( Edg ` G ) $.
+      $( Lemma 4 for ~ frgrwopreg .  In a friendship graph each vertex with
+         degree ` K ` is connected with a vertex with degree other than ` K ` .
+         This corresponds to statement 4 in [Huneke] p. 2:  "By the first
+         claim, every vertex in A is adjacent to every vertex in B.".
+         (Contributed by Alexander van der Vekens, 30-Dec-2017.)  (Revised by
+         AV, 10-May-2021.)  (Proof shortened by AV, 4-Feb-2022.) $)
+      frgrwopreglem4 $p |- ( G e. FriendGraph
+                             -> A. a e. A A. b e. B { a , b } e. E ) $=
```

Comment:
I can't see the $e hypotheses. Probably A (resp. B) is defined as the set of vertices with degree K (resp. different from K) ? Then, the statement is stronger than what the comment indicates: each vertex with degree K is connected to **any** vertex with degree different from K"